static/app/components/events/interfaces/breadcrumbs/index.tsx

```diff
@@ -32,6 +34,9 @@ type Props = Pick<React.ComponentProps<typeof Breadcrumbs>, 'route' | 'router'>
   };
   event: Event;
   organization: Organization;
+  projectSlug: string;
+  isShare?: boolean;
+  replayId?: string;
```

Comment:
could we read the `replayId` off the event inside here? fewer props to drill down that way.